docs/README.md

```diff
@@ -18,14 +18,19 @@ The documentation uses [GitHub issues](https://github.com/Sylius/Sylius/issues).
 Build
 -----
 
-If you contribute to documentation you will want to check how it looks after your changes.  
-To be able to build the documentation, install [Sphinx](http://sphinx-doc.org/).
+To test the documentation before a commit:
 
-1. [Install `pip`, Python package manager](https://pip.pypa.io/en/stable/installing/)
-2. Download documentation requirements: `$ pip install -r requirements.txt`
-3. Build the documentation:`$ sphinx-build -b html . build`
+* Install [Python](https://www.python.org/)
 
-Documentation index is `build/index.html`.
+    `$ brew install python`
+
+* Install [Sphinx](http://www.sphinx-doc.org/en/stable/)
+
+    `$ sudo pip install Sphinx`
```

Comment:
Last but not least, some Python packages may not have `pip` bundled so we should put another step between installing Python and installing Sphinx.